app_constants.sh

```diff
@@ -1,3 +1,3 @@
 env_script_location=https://raw.githubusercontent.com/fabric8-hdd/fabric8-hypothesis-core-image/master/cico_utils/setup_env.sh
-APP_NAME=fabric8-hypothesis-data-api
+APP_NAME=osio-fabric8-hypothesis-data-api
```

Comment:
please make it osio-hypothesis-data-api in-line with https://github.com/fabric8-hdd/fabric8-hypothesis-manager-api-image/blob/master/app_constants.sh#L2